packages/spacecat-shared-google-client/test/index.test.js

```diff
@@ -371,4 +371,55 @@ describe('GoogleClient', () => {
       expect(result).to.eql(testResult);
     });
   });
+
+  describe('urlInspect', () => {
+    beforeEach(() => {
+      stubSecretManager(defaultConfig);
+    });
+
+    afterEach(() => {
+      nock.cleanAll();
+      sinon.restore();
+    });
+
+    it('should inspect a valid URL', async () => {
+      const url = 'https://example.com/page';
+      const apiEndpoint = 'https://searchconsole.googleapis.com';
+      const mockResponse = { inspectionResult: 'some result' };
```

Comment:
Unit tests also play the role of documentation, so it would be useful to have here a response that has the schema of the actual response this API will return.